.github/workflows/test-monorepo.yml

```diff
@@ -0,0 +1,21 @@
+name: test-monorepo
+on:
+  push:
+    branches: main
+  pull_request:
+jobs:
+  checkout_publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: MetaMask/snaps-skunkworks
```

Comment:
Hmm. I'm wary about using real repositories here. I know this action is small, but what we introduce a bug in this action and the dry run doesn't work and we accidentally release a new version? I think if we want to add tests we should have the tests construct their own example repositories. That would also allow us to test edge cases (e.g., what happens if you have a monorepo but one of the workspace package doesn't have a `name` or `version`, etc.). Otherwise maybe it is not worth adding tests, or at least the tests we could ideally have, right now?

Reply:
this has been updated to use a ref of a version we know is already published